co-authors-plus.php

```diff
@@ -759,7 +759,7 @@ function posts_where_filter( $where, $query ) {
 						$current_user_query  = $wpdb->term_taxonomy . '.taxonomy = \'' . $this->coauthor_taxonomy . '\' AND ' . $wpdb->term_taxonomy . '.term_id = \'' . $current_coauthor_term->term_id . '\'';
 						$this->having_terms .= ' ' . $wpdb->term_taxonomy . '.term_id = \'' . $current_coauthor_term->term_id . '\' OR ';
 
-						$where = preg_replace( '/(\b(?:' . $wpdb->posts . '\.)?post_author\s*=\s*(' . get_current_user_id() . ') )/', $current_user_query . ' ', $where, -1 ); // ' . $wpdb->postmeta . '.meta_id IS NOT NULL AND}
+						$where = preg_replace( '/(\b(?:' . $wpdb->posts . '\.)?post_author\s*=\s*(' . get_current_user_id() . ') )/', $current_user_query . ' ', $where, 1 ); // ' . $wpdb->postmeta . '.meta_id IS NOT NULL AND}
```

Comment:
Do you see any other use cases or places in the code where the replacement would only need to be done once as well to prevent the SQL error from being triggered?

Reply:
Actually, yes! There are 3 places that would benefit from this fix. I've changed the additional two in https://github.com/Automattic/Co-Authors-Plus/commit/3a31a5698dfff889af4f9d7666f448880702cbc2 .